standalone/detectors/fasttrack/include/vectorclock.h

```diff
@@ -1,5 +1,7 @@
-#ifndef VECTORCLOCK_H
-#define VECTORCLOCK_H
+#ifndef VECTORCLOCK_HEADER_H
+#define VECTORCLOCK_HEADER_H 1
+#pragma once
```

Comment:
Please remove insignificant `1` and the redundant pragma since include guards already exist and are used 
```suggestion
#define VECTORCLOCK_HEADER_H
```